ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/management-api-http.yml

```diff
@@ -7,17 +7,9 @@ management_api_http:
   enable_vrfs:
     - name: default
       access_group: ACL-API
+      ipv6_access_group: ACL-API6
     - name: MGMT
       access_group: ACL-API
   protocol_https_certificate:
     certificate: <cert_string>
     private_key: <private_key>
-
-# Standard ACLs
-standard_access_lists:
```

Comment:
please add the tests in standard-acl file so we dont miss the config

Reply:
Agreed, this refactor should end up having *only* a single key in each file.